object_database/schema.py

```diff
@@ -12,30 +12,36 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
+import inspect
+
 from object_database.object import DatabaseObject, Index, Indexed
 from types import FunctionType
-from typed_python import *
```

Comment:
trying to avoid the `from X import *` style which is considered dangerous and error-prone